.gitignore

```diff
@@ -29,6 +29,7 @@
 .externalToolBuilders
 .factorypath
 .idea
+!.idea/icon.png
```

Comment:
```suggestion
```
actually dont need this if its not going to change. Most projects keep it this way.